packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1974,6 +1977,22 @@ export default class Meeting extends StatelessWebexPlugin {
       this.wirelessShare = true;
     }
 
+    if (options.meetingQuality) {
+      if (typeof options.meetingQuality === 'string') {
+        this.mediaProperties.setLocalQualityLevel(options.meetingQuality);
+        this.mediaProperties.setRemoteQualityLevel(options.meetingQuality);
+      }
+
+      if (typeof options.meetingQuality === 'object') {
+        if (options.meetingQuality.local) {
```

Comment:
These values need to be validated since the setters on `mediaProperties` don't do any validation.